lte/gateway/python/magma/subscriberdb/client.py

```diff
@@ -170,13 +223,22 @@ async def _get_all_subscribers(self) -> (SubscriberData, Digest):
             time_elapsed.total_seconds() * 1000,
         )
 
-        return subscribers, flat_digest
+        return CloudSubscribersInfo(
```

Comment:
Nit: I prefer single-line returns, even in Python code. So move the var decl to the previous line